facia/conf/routes

```diff
@@ -35,6 +35,7 @@ GET        /most-relevant-container/*path.json
 GET        /*path/show-more/*id.json                                                controllers.FaciaController.renderShowMore(path, id)
 GET        /rss                                                                     controllers.FaciaController.renderRootFrontRss()
 GET        /*path/rss                                                               controllers.FaciaController.renderFrontRss(path)
+GET        /*path/showcase                                                          controllers.FaciaController.renderFrontShowcase(path)
```

Comment:
This might also need to be added to dev-build routes

Reply:
Done in preview and dev-build.